### ISSUE TITLE
password is secret - ldap_bind(): Unable to bind to server

### ISSUE THREAD

**Issue by ken-colvin** [jotaelesalinas/laravel-adminless-ldap-auth#42]
When I do php artisan tinker and enter 

`Auth::attempt(['username' => 'einstein', 'password' => 'password'])`

I am getting this error

`PHP Warning:  ldap_bind(): Unable to bind to server: Invalid DN syntax in vendor/adldap2/adldap2/src/Connections/Ldap.php on line 346`

I am printing out the username and password and it shows as username=username, password=secret

```php
        echo sprintf("username=%s, password=%s\n", $username, $password);
        return $this->bound = ldap_bind(
            $this->connection,
            $username,
            html_entity_decode($password)
        );
```

![image](https://user-images.githubusercontent.com/8449843/84582029-59c0a900-adac-11ea-81f6-d0631acc76e8.png)

* Do you have a working version of this with ldap.forumsys.com?
* Do the defaults for LDAP_* in .env you suggest work with ldap.forumsys.com?
* Can you share your .env confing/ldap.php and config/ldap_auth.php

**Comment by jotaelesalinas** [comment]
Can you show us the value of `config('ldap_auth.identifiers.ldap.user_format')`? It should be `'uid=%s,dc=example,dc=com'`. Then, the `%s` is replaced by the username.

**Comment by ken-colvin** [comment]
Do you have this working with Laravel 6?  Can you show me a demo?  Basically, I used the defaults in your instructions, so I would assume it should work with your instructions.

**Comment by jotaelesalinas** [comment]
Here you have the old instructions, no composer package, for Laravel 6: <https://github.com/jotaelesalinas/laravel-adminless-ldap-auth/tree/d26ea52ddcc9a1336eb49a9c01469f51f8c83165>.

**Comment by Nuvelle** [comment]
I am following the guide and using 
```
Auth::attempt(['username' => 'einstein', 'password' => ''])
```

and getting the same error:

```
>>> Auth::attempt(['username' => 'einstein', 'password' => ''])
PHP Warning:  ldap_bind(): Unable to bind to server: Invalid DN syntax in C:/Users/tabbot
t/PhpstormProjects/CarrierIntergration/vendor/adldap2/adldap2/src/Connections/Ldap.php on
 line 345
PHP Warning:  ldap_bind(): Unable to bind to server: Invalid DN syntax in C:/Users/tabbot
t/PhpstormProjects/CarrierIntergration/vendor/adldap2/adldap2/src/Connections/Ldap.php on
 line 345
Adldap/Auth/BindException with message 'Invalid DN syntax'
>>>

```


This is using Laravel 7.16.1

**Comment by jotaelesalinas** [comment]
Hi @Nuvelle , did you solve the 'Invalid DN syntax' issue? What was the problem?

**Comment by Nuvelle** [comment]
No, this is still an issue for me - I am too new to Laravel to have any real chance at fixing it myself. 

Just reporting that it is an issue in the current Laravel version for me.
